packages/dev/core/src/Behaviors/Cameras/autoRotationBehavior.ts

```diff
@@ -169,6 +186,17 @@ export class AutoRotationBehavior implements Behavior<ArcRotateCamera> {
         this._lastInteractionTime = customTime ?? PrecisionDate.Now;
     }
 
+    /**
+     * Returns true if camera alpha reaches the target alpha
+     * @returns true if camera alpha reaches the target alpha
+     */
+    private _reachTargetAlpha(): boolean {
+        if (this._attachedCamera && this._targetAlpha) {
+            return this._attachedCamera.alpha <= this._targetAlpha;

```

Comment:
the alpha inspection should check if the alpha is within an epsilon of the _targetAlpha value. In this case, any time the camera's alpha is higher than the target alpha it will stop. But the user should be able to rotate the camera. if rotated a bit in the rotation direction it should start auto-rotating again.

You can use the Constants.Epsilon value to check the value is within this range.